ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/Runner.java

```diff
@@ -103,6 +110,12 @@ private Router router() {
     final Router router = Router.router(vertx);
 
     // Handler for JSON-RPC requests
+    router
+        .route()
+        .handler(
+            CorsHandler.create(buildCorsRegexFromConfig())
+                .allowedHeaders(Sets.newHashSet("*", "content-type"))
+                .allowedMethods(Sets.newHashSet(HttpMethod.POST, HttpMethod.GET)));
```

Comment:
Any need to restrict what methods this is used on? By restricting this, this will disable support for the preflight check as well that uses OPTIONS I think

Reply:
removed - making it the same as besu (basically).